lua/telescope/_extensions/file_browser/finders.lua

```diff
@@ -77,13 +84,23 @@ fb_finders.browse_files = function(opts)
       respect_gitignore = opts.respect_gitignore,
     })
   end
+
+  local file_statuses = {}
```

Comment:
Stupid nitpick, but `file_statuses` sounds wrong to me and is not meaningful once from within entry maker any longer.

`file_git_status`? not amazing but indicative of `{ file = git_status }` map

Reply:
I like how we think alike :D I had the exact same thought process. `file_git_status` just seems very wordy, I was hoping to find something shorter and crisper. I guess `file_git_status` doesn't at least lose the information that we're dealing with information from git. I will change it.